x-pack/plugins/apm/public/components/app/backend_detail_overview/backend_error_rate_chart.tsx

```diff
@@ -67,7 +71,7 @@ export function BackendErrorRateChart({ height }: { height: number }) {
         type: 'linemark',
         color: theme.eui.euiColorVis7,
         title: i18n.translate('xpack.apm.backendErrorRateChart.chartTitle', {
```

Comment:
should we update this message id as well?

Reply:
(probably not, but curious why some ids were updated and others were not)

Reply:
I started updating everything that said "error rate" but it got out of hand, so reverted most of it and mostly just kept the user facing changes.